tests.yaml

```diff
@@ -0,0 +1,239 @@
+                title: Red Hat's sanity test for MACsec support
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/macsec/sanity_check
+            redhat_tcp_keepalive:
+                title: Red Hat's tests for TCP Keep-Alive support
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/tcp/tcp_keepalive
+            redhat_udp_socket:
+                title: Red Hat's tests for UDP socket support
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/udp/udp_socket
+            redhat_bridge:
+                title: Red Hat's Ethernet bridge tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/bridge/sanity_check
+            redhat_ipsec:
+                title: Red Hat's IPSec tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/ipsec/ipsec_basic/ipsec_basic_netns
+            redhat_pmtu:
+                title: Red Hat's tests for Path MTU discovery
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/route/pmtu
+            redhat_route:
+                title: Red Hat's IP routing tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/route/route_func
+            redhat_sctp_auth_sockopts:
+                title: Red Hat's SCTP authentication socket options tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/sctp/auth/sockopts
+            redhat_sock_fuzz:
+                title: Red Hat's socket function fuzzing tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/socket/fuzz
+            redhat_geneve:
+                title: Red Hat's GENEVE sanity tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/tunnel/geneve/basic
+            redhat_gre:
+                title: Red Hat's GRE sanity tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/tunnel/gre/basic
+            redhat_vxlan:
+                title: Red Hat's VXLAN sanity tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/tunnel/vxlan/basic
+            redhat_ipvlan:
+                title: Red Hat's IPVLAN sanity tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/vnic/ipvlan/basic
+            redhat_igmp:
+                title: Red Hat's IGMP sanity tests
+                home: https://github.com/CKI-project/tests-beaker/tree/master/networking/igmp/conformance
+    security:
+        title: Security-related tests
+        tests:
+            selinux:
+                title: SELinux Regression Test Suite for the Linux Kernel
+                home: https://github.com/SELinuxProject/selinux-testsuite
+            audit:
```

Comment:
Maybe `linux-audit` would be a safer choice, as `audit` is a common word.

Reply:
Eehhh, this is all about "linux", and those are the **official** tests for **the** audit subsystem. Those are the tests which involve `kernel/audit.c` after all. So, I think this test suite deserves an unqualified name "audit".

Reply:
Well we may have tests for non-Linux systems in the future.  But yes `audit` should be fine within the current KernelCI context.